Source/Core/Common/SymbolDB.h

```diff
@@ -83,6 +85,9 @@ class SymbolDB
 	virtual Symbol *AddFunction(u32 startAddr) { return nullptr; }
 
 	void AddCompleteSymbol(const Symbol &symbol);
+	
+	void PopulateSymbolComments();
+	const std::string GetCommentFromAddr(u32 addr);
```

Comment:
Unless there is a ```static const std::string``` that represents the missing comment with an empty string or an appropriate sentence. Which is not the case here, you're right.